challenges/systems/challenge-1.md

```diff
@@ -97,6 +97,17 @@ Do not try to achieve full test coverage. This will take too long. Take two key
 components, e.g. authentication/authorization layer and networking and implement
 one or two test cases that demonstrate your approach to testing.
 
+### Dependencies
```

Comment:
Can we be even more explicit here and say that we expect cgroups to be implemented without a library?